src/base/torrentfileguard.h

```diff
@@ -29,10 +29,11 @@
 #ifndef TOFFENTFILEGURAD_H
 #define TOFFENTFILEGURAD_H
 
+#include <QObject>
 #include <QString>
-#include <QMetaType>
 
-class QMetaEnum;
+template <typename T> class CachedSettingValue;
```

Comment:
No, of course. I just mean the following: `base/bitttorrent/fwd.h`, `base/utils/fwd.h`, `base/fwd.h` (having `fwd.h` in subfolders).